test/MockSmartStore.js

```diff
@@ -522,114 +522,128 @@ var MockSmartStore = (function(window) {
         },
 
         closeCursor: function(cursorId) {
-            delete _cursors[cursorId];
+            delete this._cursors[cursorId];
         },
-
-        hookToCordova: function(cordova) {
```

Comment:
No longer built into the MockSmartStore class.
Instead it's a global function that can tie (mock) cordova to instances of MockSmartStore